client-device-connection-infinispan/src/main/java/org/eclipse/hono/deviceconnection/infinispan/client/Cache.java

```diff
@@ -67,14 +57,14 @@
     Future<V> get(K key);
 
     /**
-     * Gets a value from the cache.
+     * Remove a key/value mapping from the cache.
      *
      * @param key The key.
-     * @return A succeeded future containing the value or {@code null} if the
-     *         cache didn't contain the key yet.
-     *         A failed future if the value could not be read from the cache.
+     * @param value The value.
+     * @return {@code true} if the key was mapped to the value, {@code false}
+     *         otherwise.
```

Comment:
I would prefix the `@return` sentence with "A succeeded future containing".
I think it would be good to also mention the possibility that a failed future is returned here.

Reply:
Changed.